ios/docs/Controls/Label.md

```diff
@@ -0,0 +1,33 @@
+# Label
+## Overview
+Use `Label` to standardize text across your app.
+
+The different styles and colors of `Label` are pictured below. These styles and colors can be customized with the `attributedText` and `maxFontSize` properties. If the device's accessibility text size is changed, the `Label` text size will adjust accordingly.
+![Label.png](.attachments/Label.png)
+
+## Usage
```

Comment:
we should actually talk about the fluent font tokens which it is largely representing?